bigquery/main.go

```diff
@@ -270,7 +270,7 @@ func (bq *actualBQ) EnsureDataset(ctx context.Context, datasetName string) error
 	bq.dataset = bq.client.Dataset(datasetName)
 	_, err := bq.client.Dataset(datasetName).Metadata(ctx)
 	if err != nil {
-		log.Warningf("error obtaining dataset metadata: %v", err)
+		log.Warningf("error obtaining dataset metadata: %v\nCreating new BigQuery dataset: %v", err, datasetName)
```

Comment:
prefer `;` to `\n` for "multipart" log statements. This is because some logging systems don't handle newlines intelligently (Strackdriver logs on Cloud Run is one of those I think).